examples/developer-tools/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "developer-tools",
```

Comment:
I think we normally prefix examples with `blockly-`

Reply:
I don't want lerna to manage this right now so I explicitly left it out (this isn't really an "example" anyway, but it's not a plugin either so examples made more sense) but I can change it later if/when I do want lerna to manage it